src/OVAL/probes/unix/runlevel.c

```diff
@@ -327,6 +332,15 @@ static int is_solaris (void)
         return (stat ("/etc/release", &st)   == 0);
 }
 
+static int is_wrlinux (void)
+{
+	struct stat st;
+	int retval;
+	if ( retval = stat ("/etc/os-release", &st) != 0 ) return(retval);
```

Comment:
what is the logic behind this? WR Linux must not have /etc/os-release? Isn't the second line enough to make a good guess?